### ISSUE TITLE
[Actions]: Failed Workflow for 'Deploy Site', Docker Build Error at 'deploy-gh-pages' in the Initial Commit

### ISSUE THREAD

**Issue by WenbinWL** [sujaykundu777/devlopr-jekyll#227]
**Describe the bug**
During the initial setup and deployment using GitHub Actions, the Docker build step fails with an error indicating that the package libffi7 cannot be located. This issue prevents the successful deployment of the Jekyll site via GitHub Pages.

**To Reproduce**
1. Fork the repository.
2. Push initial commit to master branch triggering the GitHub Actions workflow.
3. The workflow fails at the deploy-gh-pages job during Docker build.
4. See error.

**Expected behavior**
The expected behavior is for the Docker container to build successfully, allowing the GitHub Actions workflow to proceed with deploying the Jekyll site to GitHub Pages without errors.

[Error Logs]
E: Unable to locate package libffi7
E: Unable to locate package libffi7

<img width="1227" alt="Screenshot 2024-07-11 at 9 11 27 PM" src="https://github.com/user-attachments/assets/d0775d97-c4ec-4b8b-b9e8-fdd941447c5f">
<img width="1843" alt="Screenshot 2024-07-11 at 9 10 57 PM" src="https://github.com/user-attachments/assets/c7556333-7e04-450d-9512-02cda58b64c0">

**Additional context**

**Comment by sujaykundu777** [comment]
Yes currently Github Pages is not supporting the latest jekyll version, trying to find a way @freyaliWL

**Comment by allitnils** [comment]
any updates on this? my understanding is that we just need to check libffi7 to libffi8?

**Comment by sujaykundu777** [comment]
Fixed it, closing it..let me know if you are facing issues with new deploys in gh-pages
